changelog/10643.misc.md

```diff
@@ -0,0 +1 @@
+Add a schema for single events so that it can be used for validation.
```

Comment:
Just needs a little bit more detail, since it's not very clear to my in which context would you need to use `EVENT_SCHEMA`  rather than `EVENTS_SCHEMA`? 😅

Reply:
haha yeah it seems kind of random, but I want to use `EVENT_SCHEMA` to validate message events in rasa x. That's why I also wanted to test the validation separately from the rasaos endpoint